### PR TITLE
Display a notification on pages that contain known issues

### PR DESCRIPTION
**Pull request by jsantell**
This could be helpful if more people start using it and explain any frustrating bugs they run into and a link to the appropriate GH issue for more information, and possibly reduce dupe bugs.

Only based off of login and codec errors at the moment -- anything else we could add?

screen shot 2016-11-02 at 10 04 31 am

### CODE REVIEW

File: app/ui/browser-modern/views/browser/decorations/notificationbar.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+const NOTIFICATION_BAR_STYLE = Style.registerStyle({
+  position: 'fixed',
+  bottom: 0,
+  left: 0,
+  right: 0,
+  height: '24px',
+  padding: '0 20px',
+  borderTop: '1px solid #bf8a01',
+  borderBottom: '1px solid #bf8a01',
```
Comment (victorporof) [inline app/ui/browser-modern/views/browser/decorations/notificationbar.jsx]: Nit: colors in theme.css

File: app/ui/browser-modern/views/browser/decorations/notificationbar.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  color: 'rgba(0,0,0,0.95)',
+  background: 'linear-gradient(#ffe13e, #ffc703)',
```
Comment (victorporof) [inline app/ui/browser-modern/views/browser/decorations/notificationbar.jsx]: Nit: backgrounds in theme.css

File: app/ui/browser-modern/views/browser/decorations/notificationbar.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  }
+
+  render() {
+    if (!this.props.message) {
+      return null;
+    }
+    return (
+      <div className={`browser-notificationbar ${NOTIFICATION_BAR_STYLE}`}>
+        <span>{this.props.message}</span>
+        <button className={NOTIFICATION_LINK_STYLE}
```
Comment (victorporof) [inline app/ui/browser-modern/views/browser/decorations/notificationbar.jsx]: Why not use a regular link here? If there's valid reasons, consider using <Btn> instead.

Comment (jsantell) [inline app/ui/browser-modern/views/browser/decorations/notificationbar.jsx]: an anchor link navigates the BrowserWindow to the page (not what we want), and using an anchor with just href='#' and an onClick has eslint complaining to use a button instead

**Comment by jsantell** [comment]
Fixed nits

**Pull request merged**